lib/config.js

```diff
@@ -141,10 +141,18 @@ function get (key, cb) {
   if (key.charAt(0) === "_") {
     return cb(new Error("---sekretz---"))
   }
-  console.log(npm.config.get(key))
+  var val = npm.config.get(key)
+  if (key.match(/umask/)) val = umask(val)
+  console.log(val)
   cb()
 }
 
+function umask (val) {
+  val = val.toString(8)
+  while (val.length < 3) val = "0" + val
```

Comment:
If you want this to look like an octal literal, it should always have a leading 0 and be 4 digits wide. Since it's a failure of ES5 strict mode to treat leading-0 literals as octals anyway, you _could_ future-proof this by adding a leading `0o`, as that's the way ES6 does it, but that's not really in keeping with the spirit of POSIX.